irohad/ametsuchi/impl/wsv_restorer_impl.cpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2018 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "wsv_restorer_impl.hpp"
+
+#include <vector>
+
```

Comment:
It is not required to explicitly call copy constructor with `emplace_back`. It is also possible to use move here: `blocks.push_back(std::move(*old_block));`